sinks/src/main/scala/com.snowplowanalytics.snowplow.eventgen/Http.scala

```diff
@@ -0,0 +1,127 @@
+/*
+ * Copyright (c) 2021-2023 Snowplow Analytics Ltd. All rights reserved.
+ *
+ * This program is licensed to you under the Apache License Version 2.0,
+ * and you may not use this file except in compliance with the Apache License Version 2.0.
+ * You may obtain a copy of the Apache License Version 2.0 at http://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the Apache License Version 2.0 is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the Apache License Version 2.0 for the specific language governing permissions and limitations there under.
+ */
+
+package com.snowplowanalytics.snowplow.eventgen
+
+import com.snowplowanalytics.snowplow.eventgen.tracker.HttpRequest
+import com.snowplowanalytics.snowplow.eventgen.tracker.HttpRequest.{Method => trackerMethod}
```

Comment:
Unusual to rename from Upper case name to lower case name.  More normal to write `{Method => TrackerMethod}`